### PR TITLE
Documentation, examples and contribution guides

### PR DESCRIPTION
**Pull request by wmai**
The purpose of this PR is to centralize our ideas and suggestions concerning the preparation of the last improvements to be made concerning the documentation before publishing the packages publicly on npm.

After discussions with the team, we decided to release an alpha version first (`0.0.1`). This version will of course not be publicly announced and it will serve as a buffer until all questions related to documentation are addressed.
For exemple, it would seem that a domain dedicated to documentation is under construction. The advantage of using this domain in our examples is to have access to "frozen" datasets ensuring us reliable results. It would therefore be a good idea to wait until this domain is ready before releasing version 1.0.0 of the packages.

Remaining to be done: 

- [x] Validate TaC/PP of CodeSandbox w/ Clémence
- [x] Make the repo `ods-dataviz-sdk` public

### CODE REVIEW

File: packages/api-client/README.md [hunk 1]
```diff
@@ -112,9 +128,9 @@ You can use that to display a better error message.
 
 The client also include a query builder.
 
-Start with one the entry point :
+Start with one the entry point:
 
-- `fromCatalog()` : query the domain catalog
+- `fromCatalog()`: query the domain catalog
```
Comment (ahervy) [inline packages/api-client/README.md:133]: I think the operation is the same for all entry points. So, we must use the same verb. Let's use "access".

File: packages/api-client/README.md [hunk 2]
```diff
@@ -24,27 +32,35 @@ yarn add @opendatasoft/api-client
...
 
-client.get(query)
-      .then((response) => console.log(response))
-      .catch((error) => console.error(error));
+const query = fromCatalog() // From the domain catalog
+    .dataset("worldcitiespop") // ... we'll use the dataset "worldcitiespop"
+    .aggregates() // ... in order to make an aggregation.
+    .where("country:'fr'") // Filtering records where country === "fr".
+    .groupBy("city, population") // Selecting the fields "city" and "population".
+    .orderBy("-population") // Sorted by the most populated cities.
```
Comment (ahervy) [inline packages/api-client/README.md:48]: I think it's clearer to mention asc/desc order where relevant. I changed the verb from "Sorted" to "Sort" to use the same verb form everywhere.
```suggestion
    .orderBy("-population") // Sort by population in descending order.
```

File: packages/api-client/README.md [hunk 3]
```diff
@@ -24,27 +32,35 @@ yarn add @opendatasoft/api-client
...
+const client = new ApiClient({ domain: "public" });
 
-client.get(query)
-      .then((response) => console.log(response))
-      .catch((error) => console.error(error));
+const query = fromCatalog() // From the domain catalog
+    .dataset("worldcitiespop") // ... we'll use the dataset "worldcitiespop"
+    .aggregates() // ... in order to make an aggregation.
+    .where("country:'fr'") // Filtering records where country === "fr".
+    .groupBy("city, population") // Selecting the fields "city" and "population".
```
Comment (ahervy) [inline packages/api-client/README.md:47]: I suggest that we change the verb from to use the same one everywhere.
```suggestion
    .groupBy("city, population") // Select the fields "city" and "population".
```

File: packages/api-client/README.md [hunk 4]
```diff
@@ -24,27 +32,35 @@ yarn add @opendatasoft/api-client
...
+// Here we initialize the Client by indicating the domain to request.
+const client = new ApiClient({ domain: "public" });
 
-client.get(query)
-      .then((response) => console.log(response))
-      .catch((error) => console.error(error));
+const query = fromCatalog() // From the domain catalog
+    .dataset("worldcitiespop") // ... we'll use the dataset "worldcitiespop"
+    .aggregates() // ... in order to make an aggregation.
+    .where("country:'fr'") // Filtering records where country === "fr".
```
Comment (ahervy) [inline packages/api-client/README.md:46]:
```suggestion
    .where("country:'fr'") // Filter records where country === "fr".
```

File: packages/visualizations/README.md [hunk 5]
```diff
@@ -1,3 +1,80 @@
-# @opendatasoft/visualizations
+# @opendatasoft/visualizations ![CI status](https://github.com/opendatasoft/ods-dataviz-sdk/workflows/CI/badge.svg)
 
-TODO: Insert badges.
+This package provides components to easily build dashboards and visualizations on top of an Opendatasoft platform. It is convenient to use those components with the [`@opendatasoft/api-client`](packages/api-client/) package which is in charge of querying the data.
```
Comment (ahervy) [inline packages/visualizations/README.md:3]: I suggest that we reword the description. We cas discuss this further.
```suggestion
This package provides components to easily build dashboards and visualizations on top of an Opendatasoft platform. An easy and convenient way to query and get data in components is to use them with the [`@opendatasoft/api-client`](packages/api-client/) package.
```

**Comment by wmai** [comment]
Thank you very much for the thorough review @ahervy ! 🙏 I will do the fixes according to your recommendation ASAP.

**Pull request merged**